samples/bluetooth/mesh/light/README.rst

```diff
@@ -145,6 +182,38 @@ Configure the Generic OnOff Server model on each element on the **Mesh Light** n
 
 Make sure to complete the configuration on each of the elements on the node to enable controlling each of the remaining three LEDs.
 
+Running DFU
+===========
+To perform a DFU, execute the following steps:
```

Comment:
```suggestion
After the sample is built with the :file:`overlay-dfu.conf` and programmed to your development kit, support for FOTA upgrades is enabled.
See FOTA in Bluetooth mesh (link) for instructions on how to perform FOTA upgrade and initiate the DFU process.
```
I guess this would replace everything else you wrote in this section.

I would recommend saving your detailed steps in case we need them later, i.e. if we want to add some details either in the existing FOTA over BLE guide or in our own mesh guide regarding FOTA upgrades.